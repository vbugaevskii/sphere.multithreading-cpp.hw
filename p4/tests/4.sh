echo blabla > file.txt
echo aaa2 > file1.txt
cat file.txt file.txt file.txt file1.txt | sort | uniq

false | true && echo 1-true
true | false || echo 2-false

