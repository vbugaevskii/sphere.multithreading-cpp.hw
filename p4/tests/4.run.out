aaa2
blabla
1-true
2-false
